### PR TITLE
Bug: Wrong index_id definition

### PR DESCRIPTION
**Pull request by StefDBA**
Index_ID = 1 is always the clustered one (B-tree and columnstore)

### CODE REVIEW

**Comment by PRMerger8** [comment]
@StefDBA : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/relational-databases/system-catalog-views/sys-indexes-transact-sql.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
...
+|**index_id**|**int**|ID of the index. **index_id** is unique only within the object.<br /><br /> 0 = Heap<br /><br /> 1 = Clustered index<br /><br /> > 1 = Nonclustered index|  
+|**type**|**tinyint**|Type of index:<br /><br /> 0 = Heap<br /><br /> 1 = Clustered b-tree<br /><br /> 2 = Nonclustered  b-tree<br /><br /> 3 = XML<br /><br /> 4 = Spatial<br /><br /> 5 = Clustered columnstore index. **Applies to**: [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] and later.<br /><br /> 6 = Nonclustered columnstore index. **Applies to**: [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)] and later.<br /><br /> 7 = Nonclustered hash index. **Applies to**: [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] and later.|
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/system-catalog-views/sys-indexes-transact-sql.md:35]:
```suggestion
|**index_id**|**int**|ID of the index. **index_id** is unique only within the object.<br /><br /> 0 = Heap<br /><br /> 1 = Clustered index<br /><br /> > 1 = Nonclustered index|  
|**type**|**tinyint**|Type of index:<br /><br /> 0 = Heap<br /><br /> 1 = Clustered rowstore (b-tree)<br /><br /> 2 = Nonclustered rowstore (b-tree)<br /><br /> 3 = XML<br /><br /> 4 = Spatial<br /><br /> 5 = Clustered columnstore index. **Applies to**: [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] and later.<br /><br /> 6 = Nonclustered columnstore index. **Applies to**: [!INCLUDE[ssSQL11](../../includes/sssql11-md.md)] and later.<br /><br /> 7 = Nonclustered hash index. **Applies to**: [!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] and later.|  
```

**Comment by PRMerger19** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger5** [comment]
@StefDBA : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Pull request merged**